README.md

```diff
@@ -3,17 +3,41 @@
 [![Build Status](https://travis-ci.com/fireworkweb/fwd.svg?branch=php)](https://travis-ci.com/fireworkweb/fwd)
 [![codecov](https://codecov.io/gh/fireworkweb/fwd/branch/php/graph/badge.svg)](https://codecov.io/gh/fireworkweb/fwd)
 
+## Description
+
+fwd it's a command line docker interface to build web applications based in `.env` files
+
+## Pre Requisites
+
+Docker
+Docker-Compose
+PHP >= 7.1.3
+
 ## Installation
 
 ```bash
 curl -L https://github.com/fireworkweb/fwd/raw/php/builds/fwd -o /usr/local/bin/fwd
 chmod +x /usr/local/bin/fwd
 ```
+Inside of your project:
 
-## Usage
+Recommended configuration of your `.env` file:
+
+```bash
+DB_CONNECTION=mysql
```

Comment:
this is nice, and should prompt us as well to check the `.env` file upon `fwd install` to make sure these entries have the proper values (create a ticket for that or provide a PR please)